pkgs/development/python-modules/spyder/default.nix

```diff
@@ -41,14 +42,14 @@
 
 buildPythonPackage rec {
   pname = "spyder";
-  version = "5.5.3";
-  format = "setuptools";
+  version = "5.5.4";
+  pyproject = true;
```

Comment:
Who in the world propagates setuptools?🤔
Could you add `setuptools` to `build-system`?

Reply:
Done